File: Samples/Verbum/how-to-implement-mail-merge-in-a-richeditcontrol-e1680-13.1.4-/VB/MailMerge/SampleData.vb
Imports System
Imports System.Text

Namespace MailMerge
    #Region "#sampledata"
    Friend Class SampleData
        Inherits System.Collections.ArrayList

        Public Sub New()
            Add(New AddresseeRecord("Andrew", "XYZ Inc.", "Tullamore Way, 21"))
            Add(New AddresseeRecord("Benny", "ABC Ltd.", "Casablanca Rd., 46"))
            Add(New AddresseeRecord("Jose", "CASA S.A.", "Paseo di Ronda, 88"))
        End Sub
    End Class
    Public Class AddresseeRecord
        Public Property Name() As String
        Public Property Company() As String
        Public Property Address() As String
        Public Sub New(ByVal fName As String, ByVal fCompany As String, ByVal fAddress As String)
            Name = fName
            Company = fCompany
            Address = fAddress
        End Sub
    End Class
    #End Region ' #sampledata
End Namespace

File: Samples/Verbum/how-to-implement-mail-merge-in-a-richeditcontrol-e1680-13.1.4-/CS/MailMerge/SampleData.cs
using System;
using System.Text;

namespace MailMerge
{
    #region #sampledata
    class SampleData : System.Collections.ArrayList
    {
        public SampleData()
        {
            Add(new AddresseeRecord("Andrew", "XYZ Inc.", "Tullamore Way, 21"));
            Add(new AddresseeRecord("Benny", "ABC Ltd.", "Casablanca Rd., 46"));
            Add(new AddresseeRecord("Jose", "CASA S.A.", "Paseo di Ronda, 88"));
        }
    }
    public class AddresseeRecord
    {
        public string Name { get; set; }
        public string Company { get; set; }
        public string Address { get; set; }
        public AddresseeRecord(string fName, string fCompany, string fAddress)
        {
            Name = fName;
            Company = fCompany;
            Address = fAddress;
        }
    }
    #endregion #sampledata
}